warehouse/integrations/vulnerabilities/models.py

```diff
@@ -0,0 +1,60 @@
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+# http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+
+from sqlalchemy import Column, ForeignKey, ForeignKeyConstraint, String, Table, orm
+from sqlalchemy.dialects.postgresql import ARRAY
+
+from warehouse import db
+
+release_vulnerabilities = Table(
+    "release_vulnerabilities",
+    db.metadata,
+    Column(
+        "release_id",
+        ForeignKey("releases.id", onupdate="CASCADE", ondelete="CASCADE"),
+        nullable=False,
+    ),
+    Column(
+        "vulnerability_source",
+        String,
+        nullable=False,
+    ),
+    Column(
+        "vulnerability_id",
+        String,
+        nullable=False,
+    ),
+    ForeignKeyConstraint(
+        ["vulnerability_source", "vulnerability_id"],
+        ["vulnerabilities.source", "vulnerabilities.id"],
+        onupdate="CASCADE",
+        ondelete="CASCADE",
+    ),
+)
+
+
+class VulnerabilityRecord(db.Model):
+
+    __tablename__ = "vulnerabilities"
+
+    source = Column(String, primary_key=True)
+    id = Column(String, primary_key=True)
+    link = Column(String)
+    aliases = Column(ARRAY(String))
```

Comment:
it's unclear to me what an alias is in this context.

Reply:
added descriptive comments for `link` and `aliases`